### PR TITLE
Formating dates to excel

### PR DESCRIPTION
**Pull request by FabienLucini**
Automatically sets the cell format to 'date' when exporting a date (yyyy-mm-dd) to Excel, using the existing `var _excelSpecials`

Credits not to me, I just saw this discussion, and wonder why it has not been PR yet ?!?!
https://datatables.net/forums/discussion/50616/what-is-the-correct-format-to-export-dates-with-html5-to-excel#Comment_134583

### CODE REVIEW

**Comment by DataTables** [comment]
Cool - thank you. But that's going to give a Javascript error due to the missing comma on:

```
{ match: /^\-?[\d,]+\.\d{2}$/,          style: 64 }  // Numbers with 2 d.p. and thousands separators
```

File: js/buttons.html5.js [hunk 1]
```diff
@@ -771,17 +772,18 @@ var excelStrings = {
...
+	{ match: /^\-?\d+\.?\d*%$/,             style: 56, fmt: function (d) { return d/100; } }, // Percent
+	{ match: /^\-?\$[\d,]+.?\d*$/,          style: 57 }, // Dollars
+	{ match: /^\-?£[\d,]+.?\d*$/,           style: 58 }, // Pounds
+	{ match: /^\-?€[\d,]+.?\d*$/,           style: 59 }, // Euros
+	{ match: /^\-?\d+$/,                    style: 65 }, // Numbers without thousand separators
+	{ match: /^\-?\d+\.\d{2}$/,             style: 66 }, // Numbers 2 d.p. without thousands separators
+	{ match: /^\([\d,]+\)$/,                style: 61, fmt: function (d) { return -1 * d.replace(/[\(\)]/g, ''); } },  // Negative numbers indicated by brackets
+	{ match: /^\([\d,]+\.\d{2}\)$/,         style: 62, fmt: function (d) { return -1 * d.replace(/[\(\)]/g, ''); } },  // Negative numbers indicated by brackets - 2d.p.
+	{ match: /^\-?[\d,]+$/,                 style: 63 }, // Numbers with thousand separators
+	{ match: /^\-?[\d,]+\.\d{2}$/,          style: 64 }  // Numbers with 2 d.p. and thousands separators
```
Comment (FabienLucini) [inline js/buttons.html5.js]:
```suggestion
	{ match: /^\-?[\d,]+\.\d{2}$/,          style: 64 },
```

**Comment by FabienLucini** [comment]
Indeed, it's fixed

**Comment by DataTables** [comment]
Lovely - thanks. Think that looks good. The CI tests will tell us soon :-)

**Pull request merged**